# Data Loading
Tujuan: Melihat dan memuat data.

## 1. Bentuk dan Ukuran Data

Pertama, kita perlu melihat seberapa banyak data yang ada.

In [6]:
import pandas as pd

In [7]:
results = pd.read_csv("../data/raw/results.csv")
former_names = pd.read_csv("../data/raw/former_names.csv")
goalscorers = pd.read_csv("../data/raw/goalscorers.csv")
shootouts = pd.read_csv("../data/raw/shootouts.csv")

In [10]:
data_data = [results, former_names, goalscorers, shootouts]

for data in data_data:
    print(data.shape)

(49477, 9)
(36, 4)
(47690, 8)
(678, 5)


Insight:
- `results` : Memiliki 49477 baris data dengan 9 kolom.
- `former_names` : Memiliki 36 baris data dengan 4 kolom.
- `goalscorers`: Memiliki 47690 baris data dengan 8 kolom.
- `shootouts`: Memiliki 678 baris data dengan 5 kolom.